docs/usage/configuration-options.md

```diff
@@ -2384,9 +2384,7 @@ e.g. if you wish to add an extra Warning to major updates:
 
 ## prBodyTemplate
 
-This setting controls which sections are rendered in the body of the pull request.
-
-The available sections are header, table, notes, changelogs, configDescription, controls, footer.
+The available sections are `header`, `table`, `warnings`, `notes`, `changelogs`, `configDescription`, `controls`, `footer`.
```

Comment:
The docs for `improveprBodyTemplate` [^docs] shows this default string:

> `"{{{header}}}{{{table}}}{{{warnings}}}{{{notes}}}{{{changelogs}}}{{{configDescription}}}{{{controls}}}{{{footer}}}"`

Does this mean that `warnings` should be added to the list of available sections?

[^docs]: https://docs.renovatebot.com/configuration-options/#prbodytemplate

Reply:
Yes, I guess so

Reply:
```suggestion
The available sections are:

- `header`
- `table`
- `warnings`
- `notes`
- `changelogs`
- `configDescription`
- `controls`
- `footer`
```

Let's convert this to a proper list while I'm at it anyways. 😄